### PR TITLE
Make the batch parameter a bit more obvious

### PR DESCRIPTION
**Pull request by 3kyro**
Make the batch size parameter a bit more obvious in the documentation of `ModifyColumnMigration`

### CODE REVIEW

File: src/Database/PostgreSQL/PQTypes/Model/Migration.hs [hunk 1]
```diff
@@ -64,13 +64,13 @@ data MigrationAction m =
   -- Name of the table that the cursor is associated with. It has to be the same as in the
   -- cursor SQL, see the second parameter.
   --
-  -- SQL that will be used for the cursor.
+  -- SQL providing a list of primary keys from the associated table that will be used for the cursor.
   --
-  -- Function that takes a list of primary keys provided by the cursor SQL and
+  -- Modification function that takes a batch of primary keys provided by the cursor SQL and
```
Comment (jsynacek) [inline src/Database/PostgreSQL/PQTypes/Model/Migration.hs:69]: Why would you point out that this function is a `Modification` function? It's called `ModifyColumnMigration` and the docs also say that it runs an arbitrary computation... To me, stating that this function modifies something is quite excessive and makes the doc a bit annoying to read.

**Review by jsynacek — COMMENTED** [review]
Otherwise, I don't really feel like your change makes the docs any more clear. What seems to be the most helpful improvement is improving the statement about that it runs in batches.
@arybczak Might want to decide this.

File: src/Database/PostgreSQL/PQTypes/Model/Migration.hs (hunk 1 above)
Comment (3kyro) [reply src/Database/PostgreSQL/PQTypes/Model/Migration.hs:69]: 👍🏽 sure it does look annoying now. I'll remove `Modification`

**Comment by 3kyro** [comment]
> Otherwise, I don't really feel like your change makes the docs any more clear. What seems to be the most helpful improvement is improving the statement about that it runs in batches. @arybczak Might want to decide this.

Clarifying that it runs in batches was the reason for this PR as it came up during a review where it wasn't clear to me.

**Review by jsynacek — APPROVED** [review]
Ok, this looks much better to me.

**Pull request merged**